### PR TITLE
Feature/graphql setup

### PR DESCRIPTION
**Pull request by Mekhak11**
Set Up GraphQL. added one query and generated.

### CODE REVIEW

**Review by Aslanyan99n — DISMISSED** [review]

File: Birthday/BirthDayAPI/Package.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    .iOS(.v12),
+    .macOS(.v10_14),
+    .tvOS(.v12),
+    .watchOS(.v5),
+  ],
+  products: [
+    .library(name: "BirthDayAPI", targets: ["BirthDayAPI"]),
+  ],
+  dependencies: [
+    .package(url: "https://github.com/apollographql/apollo-ios.git", from: "1.0.0"),
```
Comment (nellyAslanyan) [inline Birthday/BirthDayAPI/Package.swift:17]: What is the reason of using this version? there are newer

File: Birthday/Birthday/GraphQL/Client/AppoloClient.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//  Birthday
+//
+//  Created by MEKHAK GHAPANTSYAN on 21.10.24.
+//
+
+import Foundation
+import Apollo
+
+class Network {
+    static let shared = Network()
```
Comment (nellyAslanyan) [inline Birthday/Birthday/GraphQL/Client/AppoloClient.swift:13]: Please add new lines before first property and closing brace of the object. 
Same should be for all files

Comment (Mekhak11) [reply Birthday/Birthday/GraphQL/Client/AppoloClient.swift:13]: fixed.

**Review by Aslanyan99n — APPROVED** [review]

File: Birthday/Birthday/Screens/ContentView.swift [hunk 3]
```diff
@@ -19,6 +21,11 @@ struct ContentView: View {
                 .background(Color.lightGreen)
         }
         .padding()
+        .onAppear {
+          GraphQLRepository(apollo: Network.shared.apollo).performQuery(query: GetBirthDayListQuery()) { res in
```
Comment (nellyAslanyan) [inline Birthday/Birthday/Screens/ContentView.swift:25]: is this just for check?

Comment (nellyAslanyan) [reply Birthday/Birthday/Screens/ContentView.swift:25]: and try to not leave print in code :D

Comment (Mekhak11) [reply Birthday/Birthday/Screens/ContentView.swift:25]: It is just for checking for me and teammates to check if it works on their macs.

**Review by nellyAslanyan — APPROVED** [review]

**Pull request merged**